### ISSUE TITLE
The scope issue and specific physical meaning of “point_map_by_unprojection”

### ISSUE THREAD

**Issue by shanghaitech-yangjw** [facebookresearch/vggt#292]
I used the Detailed Usage provided by the author:
```python
from vggt.utils.pose_enc import pose_encoding_to_extri_intri
from vggt.utils.geometry import unproject_depth_map_to_point_map

with torch.no_grad():
    with torch.cuda.amp.autocast(dtype=dtype):
        images = images[None]  # add batch dimension
        aggregated_tokens_list, ps_idx = model.aggregator(images)
                
    # Predict Cameras
    pose_enc = model.camera_head(aggregated_tokens_list)[-1]
    # Extrinsic and intrinsic matrices, following OpenCV convention (camera from world)
    extrinsic, intrinsic = pose_encoding_to_extri_intri(pose_enc, images.shape[-2:])

    # Predict Depth Maps
    depth_map, depth_conf = model.depth_head(aggregated_tokens_list, images, ps_idx)

    # Predict Point Maps
    point_map, point_conf = model.point_head(aggregated_tokens_list, images, ps_idx)
        
    # Construct 3D Points from Depth Maps and Cameras
    # which usually leads to more accurate 3D points than point map branch
    point_map_by_unprojection = unproject_depth_map_to_point_map(depth_map.squeeze(0), 
                                                                extrinsic.squeeze(0), 
                                                                intrinsic.squeeze(0))

    # Predict Tracks
    # choose your own points to track, with shape (N, 2) for one scene
    query_points = torch.FloatTensor([[100.0, 200.0], 
                                        [60.72, 259.94]]).to(device)
    track_list, vis_score, conf_score = model.track_head(aggregated_tokens_list, images, ps_idx, query_points=query_points[None])
```
As the author mentions in the paper:

> Meanwhile, compared to directly using our estimated point maps, we found that the predictions from our depth and camera heads (i.e., unprojecting the predicted depth maps to 3D using the predicted camera parameters) yield higher accuracy. We attribute this to the benefits of decomposing a complex task (point map estimation) into simpler subproblems (depth map and camera prediction), even though camera, depth maps, and point maps are jointly supervised during training.

So i believe that the variable “point_map_by_unprojection” is the world coordinates predicted by VGGT.
The coordinates (x, y, z) of the variable “point_map_by_unprojection” that I obtained are all around 1. Is this reasonable? 

> point_map_by_unprojection range: [1.17260845  0.74922206  0.56439208]
range0 = xmax - xmin, range1 = ymax-ymin, range2 = zmax-zmin

In addition, does the variable “point_map_by_unprojection” have physical meaning and physical units, such as millimeters, centimeters, or meters?

**Comment by yyypsycheguy** [comment]
> In addition, does the variable “point_map_by_unprojection” have physical meaning and physical units, such as millimeters, centimeters, or meters?

Hi there, came across the same question at some point. Point_map_by_unprojection is the VGGT inference output, it doesn't have units, it's normalized.
